README.md

```diff
@@ -1,2 +1,78 @@
 # rust
-Docker image for Rust
+
+This is a Docker image with Rust 1.18 installed.
+
+## How to Use
+
+In order to use this image for your project, edit the `techio.yml` file and add the following lines to your project:
+
+```yaml
+    runner: techio/rust:latest
+```
```

Comment:
There are two cases:

1. the user is not expected to modify the code, put another way this is a course where people will just press play to run the code, rather than edit the code manually. In that case, it is indeed good to build the code in advance: 1 build for all is better than 1 build per user.

1. the user is expected to modify the code. Then, every time he or she plays 'Run' the code will be built. If the code resembles this example, building in advance won't change anything. But for larger projects it will be better to build in advance (even though the advantage is limited since Rust doesn't have incremental compilation yet).